Data/ProjectSettings.json

```diff
@@ -269,6 +269,16 @@
             "area": "BuildSize",
             "problem": "Managed code stripping on iOS is set to ManagedStrippingLevel.Low (or Disabled). The generated build will be larger than necessary.",
             "solution": "Set managed stripping level to Medium or High."
+        },
+        {
+          "id": 201027,
+          "description": "Player: Mipmap Stripping",
+          "type": "UnityEditor.PlayerSettings",
+          "method": "mipStripping",
+          "value": "False",
+          "area": "BuildSize",
+          "problem": "The \"Texture MipMap Stripping\" option in Player Settings is disabled. The generated build will be larger than necessary.",
```

Comment:
This is great to have added, however it could benefit from more detailed checks or description.

I.e. if the quality settings include a level for the platform that has no quality reduction then turning this on will have no effect.

I guess there is still no harm in enabling it, just will have no resultant benefit.

Also if a user modifies masterTextureLimit in code to be 0 (no reduction) and all the quality level settings have texture reduction (ie at least one mip dropped) then there will be no mip zero available to load. The code will realize this and it will user the largest mip in the asset bundle/level package, however it will change the end result on screen.

So this description could do with a caveat or two. E.g.:

"Note: Disk/download space is only saved if no quality levels on the platform use highest mip(s)."
"Note: If code drives the masterTextureLevel to a value higher than those in the quality level settings the mip will no longer be available if this is enabled"

Reply:
I changed the wording from "The generated build will be larger than necessary." to "The generated build might be larger than necessary." and I added your notes as well (see last commit)
In the future I can actually look at writing a script to analyze all the quality settings to provide a more accurate and platform-specific recommendation.